src/main/resources/assets/scripts/saml-redirect-auto-submit.js

```diff
@@ -4,4 +4,5 @@ window.setTimeout(function () {
     document.forms[0].removeAttribute("style");
 }, 5000);
 
-document.forms[0].submit();
+var submit = document.getElementById('continue-button');
+if (submit) submit.click();
```

Comment:
Worth a comment to remind us we need to click - instead of submitting directly - to get the autoLinker to automatically decorate the URL?